src/main/rules/GCI96/GCI96.json

```diff
@@ -1,4 +1,26 @@
 {
+<<<<<<< HEAD
```

Comment:
revert your modifications on all GCI96 file ... because the current PR is GCI107 and not GCI96